.gitignore

```diff
@@ -103,6 +103,7 @@ celerybeat.pid
 
 # Environments
 .env
+.docker.env
```

Comment:
in instructions these are told to be given in private

Reply:
Yeah, it's a good idea to have them in the gitignore. You don't want to accidentally commit them to a public repo, because they often will contain private keys or other sensitive info. So long as the README lists the variables that need to be in .env or otherwise set, that's good!